lib/bolt/config/options.rb

```diff
@@ -494,6 +517,7 @@ module Options
         inventoryfile
         log
         modulepath
+        plugin-cache
```

Comment:
```suggestion
```

Since this file is deprecated we shouldn't enable new features for it.

Reply:
I guess this just seemed like a nice thing to do for people who hadn't upgraded yet, since it would handle `project migrate`, but I think you're right.